docs/apv.md

```diff
@@ -511,6 +511,17 @@ Overtreding van lid 1 wordt bestraft met een straf in de 1e categorie.
 4. Een uitzondering op lid 2 is narcopolis hier op mogen gangs niet samenwerken.
 5. Bij overtreding van lid 1 en 3. Zal dit resulteren in een straf volgens de 5e categorie voor iedereen die meedeed aan het Roleplayscenario. Bij herhaling na de straf van de 5e categorie wordt er een gang strike uitgedeeld.
 
+### Artikel 55 - Appartementen
+
+1. Het uitvoeren van criminele activiteiten is verboden en valt onder het misbruiken maken van je appartement.
+2. Verduidelijking "Criminele activiteiten" in lid 1: 
```

Comment:
ik zou hier verwijzen naar het artikel bij oplichten denk ik persoonlijk dat je zeg maar 1 definitive hebt van criminele activiteiten. zodat je er geen twee krijgt straks

Reply:
true, dan ga ik het even anders verwoorden

Reply:
top je kan er denk ook wel een hyperlink in zetten haar het artikel

Reply:
ohja dat kan ook ik ga even kijken